src/status_im/contexts/communities/actions/community_options/component_spec.cljs

```diff
@@ -1,189 +1,123 @@
 (ns status-im.contexts.communities.actions.community-options.component-spec
   (:require
-    [re-frame.core :as re-frame]
     [status-im.contexts.communities.actions.community-options.view :as options]
     [test-helpers.component :as h]))
 
-(defn setup-subs
-  [subscriptions]
-  (doseq [keyval subscriptions]
-    (re-frame/reg-sub
-     (key keyval)
-     (fn [_] (val keyval)))))
-
 (h/describe "community options for bottom sheets"
+  (h/setup-restorable-re-frame)
```

Comment:
FYI: This setup function makes the dozens of warnings about subscriptions being redefined go away. Now each test starts with a blank canvas.